modules/salt/pages/custom-states.adoc

```diff
@@ -0,0 +1,151 @@
+[[custom-states]]
+= Custom Salt States
+
+You can create your own custom Salt states with {productname} as centrally managed configuration channels.
+Custom states are stored as Salt state files on the {productname} Server with a ``.sls`` extension, in the [path]``/srv/susemanager/salt/`` directory.
+
+
+
+== Create a New Custom Salt Channel
+
+You can use the {productname} {webui} to create and edit custom Salt state files.
+You must create a state channel first, with an initial state named ``init.sls``.
```

Comment:
As a complete noob, I understand this as 
```
you must create the state channel first, and then you need to
create the `init.sls` file
```
but apparently this states file will be created automatically when you create the state channel.

If that is so, I would rephrase last sentence as.
```
You must create a state channel first. This newly created channel
will contain an initial states file named ``init.sls``.
```

@juliogonzalez can you please confirm I'm not saying crap, here?

Reply:
When I went through the process, it's kind of not automatically created, though. I need to type something in the box.

Reply:
thanks for the explanation Lana